docs/api/console.mdx

```diff
@@ -778,6 +778,219 @@ _Body Parameters_
 
 ---
 
+### Create a Custom HTTP Integration (Staging)
+
+```
+POST https://staging-console.helium.wtf/api/v1/integrations
+```
+
+Creates a custom HTTP integration. Use the Console UI for JSON message templates.
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
+_Headers_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | Your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required) | _string_ | A unique human-friendly name           |
+| type (required) | _string_ | “http”             |
+| endpoint (required) | _string_ | endpoint             |
+| method (required)    | _string_ | HTTP method |
+| headers (optional) | _string_ | Map of header keys and values { “content-type”: “application/json” } |
+
```

Comment:
add row for 

url_params (optional) | map | Map for payload interpolation { "id": "{{device.id}}"}